umake/frameworks/__init__.py

```diff
@@ -340,11 +340,61 @@ def load_module(module_abs_name, main_category):
     if current_category not in BaseCategory.categories.values():
         return
     for framework_name, FrameworkClass in inspect.getmembers(module, _is_frameworkclass):
-        if FrameworkClass(current_category) is not None:
+        if FrameworkClass(category=current_category, force_loading=force_loading) is not None:
             logger.debug("Attach framework {} to {}".format(framework_name, current_category.name))
 
 
-def load_frameworks():
+def list_frameworks():
+    """ Return frameworks and category description as:
+        [
+            {
+                'category_name':
+                'category_description':
+                'is_installed': 0 (NOT_INSTALLED) or 1 (PARTIALLY_INSTALLED) or 2 (FULLY_INSTALLED)
```

Comment:
The int is an applicaiton detail, it's rather depending on the enum naming itself. So document rather the usage of BaseCategory.NOT_INSTALLED, FULL_INSTALLED, PARTIALLY_INSTALLED.